spring-rabbit/src/main/resources/org/springframework/amqp/rabbit/config/spring-rabbit-1.6.xsd

```diff
@@ -31,25 +31,41 @@
 				<xsd:annotation>
 					<xsd:documentation><![CDATA[
 						The name of the queue.  Clients can receive or listen for messages by referring to the
-						queue itself, or to its name.
+						queue itself, or to its name. When no 'name' is provided an 'AnonymousQueue' with a
+						generated name is used. Also see naming strategy.
+					]]></xsd:documentation>
+				</xsd:annotation>
+			</xsd:attribute>
+			<xsd:attribute name="naming-strategy" type="xsd:string" default="uuid">
+				<xsd:annotation>
+					<xsd:documentation><![CDATA[
+						When no 'name' is provided, a naming strategy is used to create the anonymous queue name.
+						Default (when not provided) is the String representation of a UUID. Options are
+						'uuid' - the String representation of a UUID, 'spring' - 'spring.gen-<uniqueBase64String>',
+						or a reference to an 'AnonymousQueue.NamingStrategy' implementation.
 					]]></xsd:documentation>
+					<xsd:appinfo>
+						<tool:annotation kind="ref">
+							<tool:expected-type type="java.util.Map" />
+						</tool:annotation>
+					</xsd:appinfo>
 				</xsd:annotation>
 			</xsd:attribute>
-			<xsd:attribute name="auto-delete" use="optional" type="xsd:string" default="false">
+			<xsd:attribute name="auto-delete" use="optional" type="xsd:string">
 				<xsd:annotation>
 					<xsd:documentation><![CDATA[
 						Flag indicating that an queue will be deleted when it is no longer in use, i.e. the connection that declared it is closed. Default is false.
 					]]></xsd:documentation>
 				</xsd:annotation>
 			</xsd:attribute>
-			<xsd:attribute name="exclusive" use="optional" type="xsd:string" default="false">
+			<xsd:attribute name="exclusive" use="optional" type="xsd:string">
 				<xsd:annotation>
 					<xsd:documentation><![CDATA[
 					Flag indicating that the queue is exclusive to this connection.  Default is false.
 				]]></xsd:documentation>
 				</xsd:annotation>
 			</xsd:attribute>
-			<xsd:attribute name="durable" use="optional" type="xsd:string" default="true">
```

Comment:
I had to go to a factory bean to support `spring`, `uuid` or a strategy; if you have a better idea; share it.